pkg/controller/nativeResourceWorker.go

```diff
@@ -229,7 +229,7 @@ func (ctlr *Controller) processRoutes(route *routeapi.Route, routeGroup string,
 		}
 
 		rsCfg := &ResourceConfig{}
-		rsCfg.Virtual.Partition = routeGroup
+		rsCfg.Virtual.Partition = strings.Join([]string{routeGroup, "Shared"}, "/")
```

Comment:
We will be referencing partition in other places Eg: processTLSProfilesForAS3 
Let this hold only actual partition value.

Reply:
Makes sense, I've updated it.